projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -67,18 +70,88 @@ function AccessLevelSelectorPanel( {
 	);
 }
 
+function NewsletterPrePublishSettingsPanel( {
+	accessLevel,
+	setPostMeta,
+	socialFollowers,
+	emailSubscribers,
+	paidSubscribers,
+	isModuleActive,
+} ) {
+	const { tracks } = useAnalytics();
+	const { changeStatus, isLoadingModules, isChangingStatus } = useModuleStatus( name );
+
+	const enableSubscriptionsModule = () => {
+		tracks.recordEvent( 'jetpack_editor_subscriptions_enable' );
+		return changeStatus( true );
+	};
+
+	// Subscriptions will not be triggered for a post that was already published in the past.
```

Comment:
Does this mean that the panel becomes hidden after publishing? 

If so, I think it should stay visible and allow changing the access setting, or at least allow seeing what was the setting.

Reply:
> If so, I think it should stay visible and allow changing the access setting, or at least allow seeing what was the setting.

This issue happened because the newsletter has been set already and therefore it would make little sense to change its reach.

But I definitely agree with changing the visibility of the post after publish.

Maybe we could simply change the wording and keep the panel displayed. @crisbusquets ?

@simison I'll add a separate task once this is finalized.